packages/create-cosmos-app/src/git-cca-template.ts

```diff
@@ -196,7 +196,7 @@ export const createGitApp = (repo: string, version: string) => {
         const closestPkgJson = []
             .concat(glob(join(currentDirectory, name, '**', 'package.json')))
             .reduce((shortest, current) => {
-                return current.split(sep).length < shortest.split(sep).length ? current : shortest;
```

Comment:
Make sure to use the constant sep from 'const posixPath = require('path').posix;'